docs/api.rst

```diff
@@ -27,6 +27,8 @@ Datasets
 .. autoclass:: torchani.data.BatchedANIDataset
 .. autoclass:: torchani.data.AEVCacheLoader
 .. automodule:: torchani.data.cache_aev
+.. autoclass:: torchani.data.SparseAEVCacheLoader
+.. automodule:: torchani.data.cache_sparse_aev
```

Comment:
I think the `.. automodule:: torchani.data.cache_aev` is for the file at https://github.com/aiqm/torchani/blob/master/torchani/data/cache_aev.py

So, in order to do this, you should:
- add a `cache_sparse_aev.py`, mostly copy-pasted from `cache_aev.py`
- modify the usage description in `cache_aev.py` saying it's deprecated
- in the `cache_aev` function, emit a `DeprecationWarning`

Reply:
I have added the `DeprecationWarning` and `cache_sparse_aev.py`.

Reply:
Also could you please move these above the two deprecated APIs?